test/jest/victory-area/victory-area.test.js

```diff
@@ -1,17 +1,43 @@
+import "@testing-library/jest-dom";
 import { fireEvent, render, screen } from "@testing-library/react";
-import { curveCatmullRom } from "victory-vendor/d3-shape";
 import { range } from "lodash";
 import React from "react";
 import { Area, VictoryArea } from "victory-area";
+import { VictoryChart } from "victory-chart";
+import { curveCatmullRom } from "victory-vendor/d3-shape";
 import { calculateD3Path } from "../../svg-test-helper";
 
 describe("components/victory-area", () => {
   describe("default component rendering", () => {
-    it("accepts user props", () => {
-      render(<VictoryArea data-testid="victory-area" aria-label="Chart" />);
+    it("attaches safe user props to the container component", () => {
+      render(
+        <VictoryArea
+          data-testid="victory-area"
+          aria-label="Chart"
+          unsafe-prop="test"
+        />
+      );
+
+      const container = screen.getByTestId("victory-area");
+      expect(screen.getByLabelText("Chart")).toBeDefined();
+      expect(container).not.toHaveAttribute("unsafe-prop");
```

Comment:
Isn't `toHaveAttribute` part of `jest-dom`?  I'm confused how this is working :)

Reply:
https://github.com/FormidableLabs/victory/blob/fa406575ba917fb2b84e2c6b31a5201fbd255b5d/test/jest/victory-area/victory-area.test.js#L1